libraries/bot-integration-core/src/test/java/com/microsoft/bot/integration/CloudAdapterTests.java

```diff
@@ -21,76 +22,69 @@
 import com.microsoft.bot.connector.authentication.GovernmentAuthenticationConstants;
 import com.microsoft.bot.connector.authentication.PasswordServiceClientCredentialFactory;
 import com.microsoft.bot.connector.authentication.UserTokenClient;
-import com.microsoft.bot.integration.CloudAdapter;
+import com.microsoft.bot.connector.rest.RestConnectorClient;
 import com.microsoft.bot.restclient.credentials.ServiceClientCredentials;
 import com.microsoft.bot.restclient.serializer.JacksonAdapter;
 import com.microsoft.bot.schema.Activity;
 import com.microsoft.bot.schema.ActivityTypes;
 import com.microsoft.bot.schema.ChannelAccount;
 import com.microsoft.bot.schema.ConversationAccount;
+import com.microsoft.bot.schema.ConversationParameters;
 import com.microsoft.bot.schema.ConversationReference;
-import com.microsoft.bot.schema.InvokeResponse;
+import com.microsoft.bot.schema.ConversationResourceResponse;
 import com.microsoft.bot.schema.SignInResource;
 import com.microsoft.bot.schema.TokenExchangeRequest;
 import com.microsoft.bot.schema.TokenExchangeState;
 import com.microsoft.bot.schema.TokenResponse;
 import com.microsoft.bot.schema.TokenStatus;
-import com.sun.jndi.toolkit.url.Uri;
+import okhttp3.Interceptor;
+import okhttp3.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
-import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
 import org.apache.commons.lang3.StringUtils;
-import org.apache.http.HttpStatus;
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.omg.IOP.Encoding;
+import org.mockito.Mockito;
 
 import java.io.IOException;
 import java.security.Identity;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Dictionary;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Properties;
 import java.util.concurrent.CompletableFuture;
-import java.util.stream.Stream;
-
-import static org.mockito.Mockito.*;
 
 public class CloudAdapterTests {
 
     @Test
     public void basicMessageActivity() {
         // Arrange
-        Bot botMock = mock(Bot.class);
-        when(botMock.onTurn(any (TurnContext.class))).thenReturn(CompletableFuture.completedFuture());
+        Bot botMock = Mockito.mock(Bot.class);
```

Comment:
I was not using the full namespace for Mockito methods (.mock, .when) before. If we're going to do it like this I'll change it from now on